packages/investment-pool/contracts/InvestmentPool.sol

```diff
@@ -664,11 +689,14 @@ contract InvestmentPool is IInitializableInvestmentPool, SuperAppBase, Context,
         view
         returns (bool)
     {
-        Milestone storage milestone = milestones[_milestoneId];
+        Milestone memory milestone = milestones[_milestoneId];
```

Comment:
Why do we swap for memory here?

Reply:
We only use milestone variable to get data, so memory is enough for this function.